### PR TITLE
Remove project contents

### PR DESCRIPTION
**Pull request by marten**
Describe your change here.

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

File: app/services/media_storage.rb [hunk 1]
```diff
@@ -2,6 +2,9 @@
 require 'media_storage/aws_adapter'
 
 module MediaStorage
+  class EmptyPathError < StandardError; end
```
[the remark this answers is not in the document]
Comment (marten) [reply app/services/media_storage.rb]: Unrelated but encountered an error with them while running the suite, dependent on rspec order seed: https://github.com/zooniverse/Panoptes/pull/3077/commits/d663ac900d440ca18de0feeb8b3caeba146ee03c

**Comment by marten** [comment]
Nope, not WIP. Don't know if we want to merge it now or wait till I'm back?
